src/v2/Artsy/Router/__tests__/Boot.jest.tsx

```diff
@@ -55,4 +55,10 @@ describe("Boot", () => {
   it("injects Grid", () => {
     expect(mount(<Boot {...bootProps} />).find("Grid").length).toEqual(1)
   })
+
+  it("injects RelayEnvironmentProvider", () => {
+    expect(
+      mount(<Boot {...bootProps} />).find("RelayEnvironmentProvider").length
+    ).toEqual(1)
```

Comment:
thought: this test made me question life and code 😛. I've never thought of looking to see if a there exists exactly one context provider of a specific type. interesting! 👍

Reply:
well its more like just a truthy check and a lock so that folks don't mess with boot later unexpectedly. Could have easily done `.exists()` or `isDefined()`, etc.